src/components/LabeledSelect.css

```
@@ -72,6 +72,23 @@
+.labeled-select--hide-label .labeled-select-label {
+  gap: 0;
+}
+
+.labeled-select--hide-label .labeled-select-label-text {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.7.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)